main.js

```diff
@@ -0,0 +1,132 @@
+//multiply number vs 1 to 10
+
+const multiplyNum = num => {
+    console.log('In bang nhan:')
+    for (let i = 1; i < 11; i++) {
+        console.log(num + ' x ' + i + ' = ' + ' ' + num * i)
+    }
+}
+
+//find even number
+
+const findEvenNum = num => {
+    if (num >=1 && num <=30) {
+        for (let i = 1; i < 31; i++) {
+            if (i % 2 === 0) {
+                console.log(i)
+            }
+            if (i >= num) break
+        }
```

Comment:
Quá nhiều if
Đọc lại đề: `Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30
In các số chẵn từ 1 đến n
Vd n = 6: in ra: 2 4 6`

```javascript
  if (num < 1 || i > 30) return;

  for (let i = 1; i < num; i++) {
    if (i % 2 === 0) {
      console.log(i)
    }
  }
```